frontend/apps/reader/modules/readerlink.lua

```diff
@@ -104,6 +105,7 @@ function ReaderLink:addToMainMenu(menu_items)
                     G_reader_settings:saveSetting("swipe_to_go_back",
                         not isSwipeToGoBackEnabled())
                 end,
+                help_text = _([[Swipe to the right to go back to the previous location after you have followed a link. When the location stack is empty, swiping to the right takes you normally to previous page.]]),
```

Comment:
Just `takes you to the previous page` should suffice (without normally, but with the determiner)